plugins/sigma.layout.fruchtermanReingold/sigma.layout.fruchtermanReingold.js

```diff
@@ -0,0 +1,285 @@
+;(function(undefined) {
+  'use strict';
+
+  if (typeof sigma === 'undefined')
+    throw new Error('sigma is not declared');
+
+  // Initialize package:
+  sigma.utils.pkg('sigma.layouts.fruchtermanReingold');
+
+  /**
+   * Sigma Fruchterman-Reingold
+   * ===============================
+   *
+   * Author: Sébastien Heymann @ Linkurious
+   * Version: 0.1
+   */
+
+
+  var settings = {
+    autoArea: true,
+    area: 1,
+    gravity: 10,
+    speed: 0.1,
+    maxIterations: 1000
+  };
+
+  /**
+   * Event emitter Object
+   * ------------------
+   */
+  var eventEmitter = {};
+  sigma.classes.dispatcher.extend(eventEmitter);
+
+  /**
+   * Fruchterman Object
+   * ------------------
+   */
+  function FruchtermanReingold() {
+  };
+
+  FruchtermanReingold.prototype.init = function (sigInst, options) {
+    options = options || {};
+
+    // Properties
+    this.sigInst = sigInst;
+    this.graph = this.sigInst.graph;
+    this.config = sigma.utils.extend(options, settings);
+    this.easing = options.easing;
+    this.duration = options.duration;
+
+    if (!sigma.plugins || typeof sigma.plugins.animate === 'undefined')
+      throw new Error('sigma.plugins.animate is not declared');
+
+    // State
+    this.started = false;
+    this.running = false;
+
+    // Binding on kill to properly terminate layout when parent is killed
+    this.sigInst.bind('kill', function() {
+      eventEmitter = {};
+      sigma.classes.dispatcher.extend(eventEmitter);
+      sigma.layouts.fruchterman.stop();
+    });
+  };
+
+  FruchtermanReingold.prototype.atomicGo = function () {
+    if (!this.running || this.iterCount < 1) return false;
+
+    var self = this,
+        nodes = this.graph.nodes(),
+        edges = this.graph.edges(),
+        i,
+        j,
+        n,
+        n2,
+        e,
```

Comment:
`e` is conventionnaly used in the Sigma source code for edges, so I prefer to keep it.